### ISSUE TITLE
Incompatible with React Native ^0.15.0

### ISSUE THREAD

**Issue by kauffecup** [tomauty/react-native-chart#33]
• Uses outdated requires
• Uses `createNativeComponentClass` instead of `requireNativeComponent` and `setNativeProps`
• Uses `NativeMethodsMixin`

Throws errors like:

``` sh
2015-12-15 10:20:02.399 [error][tid:main] Unable to resolve module ReactPropTypes
from /Workspace/NameOfProject/node_modules/react-native-chart/Chart.ios.js:
Invalid directory /node_modules/ReactPropTypes

2015-12-15 10:24:31.510 [error][tid:main] Unable to resolve module
ReactNativeViewAttributes from /Workspace/NameOfProject/node_modules/
react-native-chart/Chart.ios.js:
Invalid directory /node_modules/ReactNativeViewAttributes
```

**Comment by L-Jovi** [comment]
the same issue ..

**Comment by kauffecup** [comment]
fixed with PR #38
